packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.tsx

```diff
@@ -20,7 +20,8 @@ const TransferFormAccountCard: React.FC<TProps> = ({ account, type = 'modal' })
     const isModal = type === 'modal';
 
     const hasPlatformStatus =
-        account?.status === TRADING_PLATFORM_STATUS.UNAVAILABLE || TRADING_PLATFORM_STATUS.MAINTENANCE;
+        account?.status === TRADING_PLATFORM_STATUS.UNAVAILABLE ||
+        account?.status === TRADING_PLATFORM_STATUS.MAINTENANCE;
```

Comment:
Fix for badge which is always displaying in transfer form:
<img width="704" alt="Bildschirmfoto 2024-11-06 um 10 47 49 AM" src="https://github.com/user-attachments/assets/96b34ef7-084a-46c5-8518-d0d7981a2b55">